interface.py

```diff
@@ -15,110 +15,113 @@ def __init__(self, dealt_cards, robot_data):
                 self.cursor_index += 1
 
-
     def cursor_index_minus(self):
         """
         Change selecting cursor position to the previous one.
         """
-        if self.indicator == False:
+        if not self.indicator:
             if self.cursor_index > 0:
                 self.cursor_index -= 1
 
-
     def switch_power_down(self):
         """
         Switch power down status between True and False.
         When it is True the Robot doesn't play this round.
         """
-        if self.indicator == False:
-            if self.power_down == False:
+        if not self.indicator:
+            if not self.power_down:
                 self.power_down = True
             else:
                 self.power_down = False
 
-
     def confirm_selection(self):
         """
-        When indicator is False the player can choose cards and switch Power Down
-        When is True the player ended the selection of cards
+        When indicator is False the player can choose cards and switch Power Down.
+        When is True the player ended the selection of cards.
         """
         self.indicator = True
 
 
 def create_card_pack():
+    """
+    Create pack of cards: 42 movement cards and 42 rotation cards
+    with different values and priorities.
+    Return shuffled card pack.
+    """
     movement_cards = [(-1, 6, 250),
-                    (1, 18, 300),
-                    (2, 12, 400),
-                    (3, 6, 500),
-                    ]
+                      (1, 18, 300),
+                      (2, 12, 400),
+                      (3, 6, 500),
+                      ]
     rotation_cards = [(Rotation.U_TURN, 6, 50),
-                    (Rotation.LEFT, 18, 100),
-                    (Rotation.RIGHT, 18, 200),
-                    ]
+                      (Rotation.LEFT, 18, 100),
+                      (Rotation.RIGHT, 18, 200),
+                      ]
     card_pack = []
 
     for movement, cards_count, first_number in movement_cards:
         for i in range(cards_count):
-            card_pack.append(MovementCard(first_number + i*5, movement)) # [MovementCard(690, -1)...][]
+            # [MovementCard(690, -1)...][]
+            card_pack.append(MovementCard(first_number + i*5, movement))
 
     for rotation, cards_count, first_number in rotation_cards:
         for i in range(cards_count):
-            card_pack.append(RotationCard(first_number + i*5, rotation)) # [RotationCard(865, Rotation.LEFT)....]
+            # [RotationCard(865, Rotation.LEFT)....]
+            card_pack.append(RotationCard(first_number + i*5, rotation))
     shuffle(card_pack)
-    #print(card_pack)
     return card_pack
 
 
 card_pack = create_card_pack()
 
 
 def get_dealt_cards(card_pack):
-    # maximum number of cards is 9
-    # robot's damages reduces the count dealt cards
+    """
+    Maximum number of cards is 9.
+    Robot's damages reduces the count of dealt cards - each damage one card.
+    """
```

Comment:
Tohle není popis funkce (cílený na toho, kdo funkci chce zavolat), ale komentáře (vysvětlení kódu nebo detailů). Původně to bylo správnější.

Reply:
Díky za upozornění, předělám.